src/backend/people/settings.py

```diff
@@ -335,7 +335,7 @@ class Base(Configuration):
         {}, environ_name="OIDC_AUTH_REQUEST_EXTRA_PARAMS", environ_prefix=None
     )
     OIDC_RP_SCOPES = values.Value(
-        "openid email", environ_name="OIDC_RP_SCOPES", environ_prefix=None
+        "openid email siret", environ_name="OIDC_RP_SCOPES", environ_prefix=None
```

Comment:
should probably be set in the argocd conf only to make the os project more generic?